docker/release/Dockerfile

```
@@ -1,11 +1,15 @@
 FROM ubuntu:18.04
 
-RUN apt-get update
+RUN apt-get update; \
+    apt install -y software-properties-common; \
+    add-apt-repository ppa:ubuntu-toolchain-r/test
```

```suggestion
    add-apt-repository ppa:ubuntu-toolchain-r/ppa
```
I am afraid test can contain breaking changes.  
https://wiki.ubuntu.com/ToolChain
>PPA packages
Updated toolchain packages can be found in two PPAs.
https://launchpad.net/~ubuntu-toolchain-r/+archive/ppa hosts updates from release branches (e.g. Ubuntu 14.04 LTS released with gcc-4.8.2 and the PPA has gcc-4.8.3).
https://launchpad.net/~ubuntu-toolchain-r/+archive/test hosts newer upstream versions (e.g. gcc-4.9 is found here for Ubuntu 14.04 LTS).

---

`test` PPA is used in Develop Dockerfile, I suggest to use the same here.

---

Ok